pallets/ajuna-awesome-avatars/src/migration/v5.rs

```diff
@@ -120,11 +127,13 @@ impl<T: Config> OnRuntimeUpgrade for MigrateToV5<T> {
 		let current_version = Pallet::<T>::current_storage_version();
 		let onchain_version = Pallet::<T>::on_chain_storage_version();
 		if onchain_version == 4 && current_version == 5 {
+			let current_season_id = CurrentSeasonId::<T>::get();
+
 			let _ = CurrentSeasonStatus::<T>::translate::<OldSeasonStatus, _>(|maybe_old_value| {
 				maybe_old_value.map(|old_value| {
 					log::info!(target: LOG_TARGET, "Migrated current season status");
 					let mut new_value = old_value.migrate_to_v5();
-					new_value.season_id = CurrentSeasonId::<T>::get();
+					new_value.season_id = current_season_id;
```

Comment:
Do we need these changes? If not, can we revert back

Reply:
You are right, I've reverted the changes!